packages/dev/loaders/src/glTF/2.0/Extensions/KHR_animation_pointer.map.ts

```diff
@@ -0,0 +1,510 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { Animation } from "core/Animations/animation";
+import type { AnimationGroup } from "core/Animations/animationGroup";
+import { Quaternion, Vector3, Matrix } from "core/Maths/math.vector";
+import { Color3 } from "core/Maths/math.color";
+import type { IGLTF } from "../glTFLoaderInterfaces";
+import { Material } from "core/Materials/material";
+import type { IAnimatable } from "core/Animations/animatable.interface";
+import type { Mesh } from "core/Meshes/mesh";
+import type { Nullable } from "core/types";
+
+export type GetGltfNodeTargetFn = (source: IGLTF, indices: string) => any;
+export type GetValueFn = (target: any, source: Float32Array, offset: number, scale?: number) => any;

```

Comment:
I'm not sure why these are exported. They are not used outside of this file, right?

Reply:
The first one is used as delegate type into the pointer parsing to access the object. i remove the second.